docs/das-cli-users-guide.md

```diff
@@ -0,0 +1,450 @@
+
+Example: To restart the Inference Agent, you would run:
+
+1. Linux users
+```bash
+das-cli inference-agent restart
+```
+2. macOS/Windows users. **Choose one of the options below.**
+```bash
+python das_cli.py inference-agent restart --remote --host 192.168.0.10 --user ubuntu --key-file ~/.ssh/id_rsa
+
+python das_cli.py inference-agent restart --remote --host 192.168.0.10 --user ubuntu --password my_p@ssword
+```
+
+### Managing Databases
+
+This command group controls the core storage components of the DAS (Redis and MongoDB).
+
+- **start | stop | restart:** These subcommands manage the lifecycle of the database containers. start launches them, stop shuts them down, and restart does both in sequence.
+- **count-atoms:** A diagnostic tool. After loading a MeTTa file, use this command to connect to the databases and display the total number of atoms stored. This is the best way to verify that your metta load command was successful.
+
+Example:
+
+1. Linux users
+```bash
+das-cli db count-atoms
+```
+2. macOS/Windows users. **Choose one of the options below.**
+```bash
+python das_cli.py db count-atoms --remote --host 192.168.0.10 --user ubuntu --key-file ~/.ssh/id_rsa
+
+python das_cli.py db count-atoms --remote --host 192.168.0.10 --user ubuntu --password my_p@ssword
+```
+
+If you want more details about the knowledge base, you can pass the `--verbose` flag.
+
+```bash
+das-cli db count-atoms --verbose
+python das_cli.py db count-atoms --verbose --remote --host 192.168.0.10 --user ubuntu --key-file ~/.ssh/id_rsa
+```
+
+### Managing MeTTa Files
+
+This group handles the knowledge base files that feed the **DAS**.
+
+- **check**: Validates the syntax of a .metta file without loading it. It is highly recommended to run this before any load operation to catch errors early.
+- **load**: Parses a .metta file and imports its atoms into the databases.
+
+### Managing Configuration
+
+This group manages the settings `das-cli` uses to connect to services.
+
+- **set**: Launches the interactive prompt to configure all parameters.
+- **list (or ls)**: Displays all the current configuration values in your terminal, which is useful for a quick check without going through the interactive setup.
+
+### Monitoring with Logs
+
+The logs command is tool for debugging and monitoring the health of your services. You can view the logs for any running service, including all agents and databases.
+
+The general command pattern is: `das-cli logs <service-name>`
+
+Example: To debug a potential issue with Redis, you can view its logs:
+
+1. Linux users
+```bash
+das-cli logs attention-broker
+```
+2. macOS/Windows users. **Choose one of the options below.**
+```bash
+python das_cli.py logs attention-broker --remote --host 192.168.0.10 --user ubuntu --key-file ~/.ssh/id_rsa
+
+python das_cli.py logs attention-broker --remote --host 192.168.0.10 --user ubuntu --password my_p@ssword
+```
+
+---
+
+## Troubleshooting and Tips
```

Comment:
You should add more cases here or remove this section. The way it is it's very poor.